lib/ui/widgets/components/app_button_tiny.dart

```diff
@@ -313,7 +316,12 @@ class _IconButton extends ConsumerWidget {
           AutoSizeText(
             buttonText,
             textAlign: TextAlign.center,
-            style: theme.textStyleSize12W400EquinoxMainButtonLabel,
+            style: theme.textStyleSize12W400EquinoxMainButtonLabel.copyWith(
+              color: onPressed == null
```

Comment:
Why the size of the text is not similar between disabled and enabled buttons ?

Reply:
My bad